bake/compose.go

```diff
@@ -14,34 +16,22 @@ import (
 // errComposeInvalid is returned when a compose file is invalid
 var errComposeInvalid = errors.New("invalid compose file")
 
-func parseCompose(dt []byte) (*compose.Project, error) {
-	return loader.Load(compose.ConfigDetails{
+func ParseCompose(dt []byte, envs map[string]string) (*Config, error) {
+	if wd, err := os.Getwd(); err == nil {
```

Comment:
Looks like this suits better in `ParseComposeFile`. `envs` is also passed to this function so shouldn't be extended. But `ParseComposeFile` already reads env from the host.